app/models/move.rb

```diff
@@ -12,6 +12,12 @@ class Move < ApplicationRecord
     prison_transfer: 'prison_transfer'
   }
 
+  enum cancellation_reason: {
+    created_by_mistake: 'created_by_mistake',
```

Comment:
Sorry to be a pain. Have had a chat with @lmoney-moj about the option names and I think we should match them to the language we use in the interface to avoid any confusion when looking at the data from the API.

Could we use these reasons as the enum options please:
- `made_in_error`
- `supplier_declined_to_move`
- `other`